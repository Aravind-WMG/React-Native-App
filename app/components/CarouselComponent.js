import React from 'react';
import {Text, View , StyleSheet, Dimensions, Image} from 'react-native';
import Carousel from 'react-native-snap-carousel';

const { height, width } = Dimensions.get('window');

export class CarouselComponent extends React.Component{
    constructor(props){
        super(props);
    }
    state = {
        images: [{"id":"1","pdt_name":"Kidz Jeans"},
        {"id":"2","pdt_name":"Mens Jeans"},
        {"id":"3","pdt_name":"Womens Jeans"},
        {"id":"4","pdt_name":"Junior Jeans"},
        {"id":"5","pdt_name":"Pluz size Jeans"}
    ]
    };

    renderItem = ({item, index}) => {
        return (
            <View style={styles.carouselWrapper}>
                <Image style={styles.logoStyle} source={require('../images/jeans-carousel.jpg')} />
                <Text style={styles.logoTitle}>{item.pdt_name}</Text>
            </View>
        );
    }

    render(){
        return(
            <View style={styles.mainWrapper}>
                <Text style={styles.carouselTitle}>Shop By Fit</Text>
                <Carousel
                      inactiveSlideOpacity={1}
                      inactiveSlideScale={0.95}
                      firstItem={0}
                      sliderWidth={width}
                      itemWidth={width/2.5}
                      activeSlideAlignment={'start'}
                      activeAnimationType={'spring'}
                      data={this.state.images}
                      renderItem={this.renderItem}
                    />
            </View>
 
        )
    }
}

const styles = StyleSheet.create({
    carouselWrapper:{
        flex:1,
        flexDirection:'row',
        alignItems:'center',
        backgroundColor:'#fff',
        borderRadius:5,
        borderWidth: 1,
        borderColor: '#000',
    },
    mainWrapper:{
        paddingVertical: 20
    },
    carouselTitle: {
      color:'#000',
      fontSize: 24,
      fontWeight: 'bold',
      textAlign: 'center', 
      paddingBottom:15
    },
    logoStyle: {
        width: 50,
        height: 50,
        borderRadius:10,
    },
    logoTitle:{
        fontSize: 16,
        fontWeight: 'bold',
        textAlign: 'center',
        width:75
    }
  });