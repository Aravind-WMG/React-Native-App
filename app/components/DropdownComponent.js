import React, { Component } from 'react';
import { StyleSheet, View, Text, TouchableOpacity,Alert } from 'react-native';
import ModalDropdown from 'react-native-modal-dropdown';
import Icon from 'react-native-vector-icons/MaterialIcons';
import { sampleDropdownOptions, sampleProductDetail } from '../data/DummyData';

export class DropdownComponent extends Component {
    constructor(props) {
        super(props);
        this.state = {
            dataSource: sampleProductDetail,
            dropdowList:sampleDropdownOptions
        }
    }

    dropdownRenderRow(rowData, rowID, highlighted) {
        if (rowID == 1) {
            return (
                <View style={styles.dropdownStyle}>
                    <Icon name="store" size={30} color='#000' />
                    <Text style={styles.dropdownTextStyle}>
                    <Text style={{fontWeight: "bold"}}> {rowData} </Text>
                    ({this.state.dataSource[0].product.bopos}){"\n"}
                    at <Icon name="star" size={20} color='#000' /> StoreName
              </Text>
                </View>
            );
        }
        else if (rowID == 2) {
            return (
                <View style={styles.dropdownStyle}>
                    <Icon name="store" size={30} color='#000' />
                    <Text style={styles.dropdownTextStyle}>
                    <Text style={{fontWeight: "bold"}}> {rowData} </Text>
                    Free ({this.state.dataSource[0].product.boss})
              </Text>
                </View>
            );
        }
        else if (rowID == 3) {
            return (
                <View style={styles.dropdownStyle}>
                    <Icon name="local-shipping" size={30} color='#000' />
                    <Text style={styles.dropdownTextStyle}>
                    <Text style={{fontWeight: "bold"}}> {rowData} </Text>
                     ({this.state.dataSource[0].product.ship}){"\n"}
                        Free at $75
              </Text>
                </View>
            );
        }
        else {
            return (
                <View style={styles.dropdownStyle}>
                    <Text style={styles.dropdownTextStyle}>
                        {rowData} ({this.state.dataSource[0].product.product_count})
              </Text>
                </View>
            );
        }
    }
    onClickDropDown=(selectedVal)=>{
        //Alert.alert(selectedVal)
        this.props.onDropDownSelect(selectedVal);
        //return selectedVal;
    }
    dropdownRenderSeparator(sectionID, rowID, adjacentRowHighlighted) {
        if (rowID == 4) return true;
        let key = rowID;
        return (
            <View style={styles.dropdownSeparator} key={key} />);
    }

    componentDidMount() {
        this.dropdowList = sampleDropdownOptions;
        this.dropdowListDefault = "Pickup & Shipping";
        this.setState({
            dropdowList: this.dropdowList,
            dropdowListDefault: this.dropdowListDefault
        }, function () {
            // In this block you can do something with new state.
        });
    }

    render() {
        return (
            <View>
                <Text style={styles.productHeader}>{this.state.dataSource[0].product.search_term} ({this.state.dataSource[0].product.product_count})</Text>
                <TouchableOpacity>
                    <View style={{ height: 40, alignItems: 'center', flexDirection: 'row' }}>
                        <ModalDropdown ref={(el) => { this.dropDown = el }}
                            options={this.state.dropdowList}
                            defaultValue={this.state.dropdowListDefault}
                            style={styles.buttonStyle}
                            textStyle={styles.textStyle}
                            dropdownStyle={styles.dropdownStyle}
                            renderRow={this.dropdownRenderRow.bind(this)}
                            renderSeparator={this.dropdownRenderSeparator.bind(this)}
                            dropdownTextStyle={styles.dropdownTextStyle}
                            onSelect = {(index,value)=>{this.onClickDropDown(value)}}
                        />
                        <View style={{ position: "absolute", right: 20, top: 10 }}><Text>â–¼</Text></View>
                    </View>
                </TouchableOpacity>
            </View>
        );
    }


}

const styles = StyleSheet.create({
    buttonStyle: {
        width: '98%',
        paddingVertical: 10,
        backgroundColor: '#bec1c6',
    },
    textStyle: {
        fontWeight: 'bold',
        textAlign: 'left',
        fontSize: 16,
        paddingLeft: 5,
        color: '#000'
    },
    dropdownStyle: {
        width: '94%',
        flexDirection: 'row',
        alignItems: 'center',
        marginTop: 10,
        height: 'auto'
    },
    dropdownTextStyle: {
        textAlign: 'left',
        fontSize: 16,
        paddingVertical: 10,
        color: '#000',
        paddingLeft: 10,
        
    },
    dropdownContainer: {
        marginTop: 25,
    },
    productHeader: {
        fontWeight: 'bold',
        fontSize: 18,
        paddingBottom: 5,
        color: '#000',
    },
    dropdownSeparator: {
        height: 1,
        backgroundColor: '#000',
    }
})