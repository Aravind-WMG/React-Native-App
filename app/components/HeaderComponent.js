import React from 'react';
import {Text, View, StyleSheet, Image,TouchableOpacity } from 'react-native';
import Icon from 'react-native-vector-icons/MaterialIcons';

export class HeaderComponent extends React.Component{
    constructor(props){
        super(props);
    }
    render(){
        return(
            <View style={styles.navBar}>
                <View style={styles.navBarLeft}>
                    <TouchableOpacity>
                        <Icon name="menu" size={40}/>
                    </TouchableOpacity>
                    <Image 
                        source={require('../images/logo_kohls.png')} 
                        style={{width:180,height:27,backgroundColor:'#000'}}
                    />
                </View>
                <View style={styles.navBarRight}>
                    <Text style={styles.userName}>Hi, UserName</Text>
                    <TouchableOpacity>
                        <Icon name="account-circle" size={40}/>
                    </TouchableOpacity>
                    <TouchableOpacity>
                        <Icon name="shopping-cart" size={40}/>
                    </TouchableOpacity>
                </View>
            </View>
        )
    }
}
const styles = StyleSheet.create({
    navBar: {
      height: 75,
      backgroundColor: '#fff',
      elevation:3,
      flexDirection:'row',
      alignItems:'center',
      
    },
    navBarLeft: {
        flexDirection:'row',
        justifyContent:'space-between',
        alignItems:'center',
    },
    navBarRight: {
        flexDirection:'row',
        alignItems:'center',
        paddingLeft:20,
        justifyContent:'space-between',
    },
    userName:{
        color:'#000',
        fontSize: 14,
        fontWeight: 'bold',
    }
  });