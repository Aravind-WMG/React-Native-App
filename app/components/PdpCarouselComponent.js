import React from 'react';
import { Text, View, StyleSheet, Dimensions, Image, Alert } from 'react-native';
import Carousel,{ Pagination } from 'react-native-snap-carousel';
import { sampleSingleProductDetail } from '../data/DummyData';
import Imageprogressive from 'react-native-image-progress';
import ProgressBar from 'react-native-progress/Bar';

//const { height, width } = Dimensions.get('window');

export class PdpCarouselComponent extends React.Component {
    constructor(props) {
        super(props);
        this.state = {
            carouselData: sampleSingleProductDetail,
            activeSlide:0,
            viewport: {
                width: Dimensions.get('window').width,
                height: Dimensions.get('window').height
            }
        }
    }

    componentDidMount() {
        //Alert.alert("Hey");
    }
    get pagination () {
        const { carouselData, activeSlide } = this.state;
        return (
            <Pagination
              dotsLength={carouselData[0].singlePdtDetail.visualNavTile.length}
              activeDotIndex={activeSlide}
              dotStyle={{
                  width: 10,
                  height: 10,
                  borderRadius: 5,
                  marginHorizontal: 1,
                  backgroundColor: 'rgba(0, 0, 0, 0.92)'
              }}
              inactiveDotStyle={{
                  // Define styles for inactive dots here
              }}
              inactiveDotOpacity={0.4}
              inactiveDotScale={1}
            />
        );
    }
    renderItem = ({ item, index }) => {
        return (
            <View style={styles.carouselWrapper}>
                <Imageprogressive
                    style={styles.logoStyle}
                    source={{ uri: item.imageUrl }}
                    indicator={ProgressBar.Pie}
                    indicatorProps={{
                        size: 40,
                        borderWidth: 0,
                        color: 'rgba(150, 150, 150, 1)',
                        unfilledColor: 'rgba(200, 200, 200, 0.2)'
                    }}
                />
            </View>
        );
    }

    render() {
        return (
            <View style={styles.mainWrapper} 
                onLayout={() => {
                    this.setState({
                        viewport: {
                            width: Dimensions.get('window').width,
                            height: Dimensions.get('window').height
                        }
                    });
                }}>
                <Carousel
                    inactiveSlideOpacity={1}
                    inactiveSlideScale={1}
                    firstItem={0}
                    sliderWidth={this.state.viewport.width}
                    itemWidth={this.state.viewport.width}
                    activeSlideAlignment={'start'}
                    data={this.state.carouselData[0].singlePdtDetail.visualNavTile}
                    renderItem={this.renderItem}
                    onSnapToItem={(index) => this.setState({ activeSlide: index }) }
                />
                { this.pagination }
            </View>

        )
    }
}

const styles = StyleSheet.create({
    carouselWrapper: {
        flex: 1,
        flexDirection: 'row',
        alignItems: 'center',
        backgroundColor: '#fff',
        justifyContent:'center'
    },
    mainWrapper: {
        paddingVertical: 0,
        flex: 1
    },
    logoStyle: {
      width:320,
      height:320
    },
    
});