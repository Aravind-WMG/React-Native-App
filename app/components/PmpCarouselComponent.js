import React from 'react';
import { Text, View, StyleSheet, Dimensions, Image, Alert } from 'react-native';
import Carousel from 'react-native-snap-carousel';
import { sampleCarouselData } from '../data/DummyData';
import Imageprogressive from 'react-native-image-progress';
import ProgressBar from 'react-native-progress/Bar';

//const { height, width } = Dimensions.get('window');

export class PmpCarouselComponent extends React.Component {
    constructor(props) {
        super(props);
        this.state = {
            carouselData: sampleCarouselData,
            viewport: {
                width: Dimensions.get('window').width,
                height: Dimensions.get('window').height
            }
        }
    }

    componentDidMount() {
        //Alert.alert("Hey");
    }

    renderItem = ({ item, index }) => {
        return (
            <View style={styles.carouselWrapper}>
                <Imageprogressive
                    style={styles.logoStyle}
                    source={{ uri: item.imageUrl }}
                    indicator={ProgressBar.Pie}
                    resizeMode='cover'
                    indicatorProps={{
                        size: 20,
                        borderWidth: 0,
                        color: 'rgba(150, 150, 150, 1)',
                        unfilledColor: 'rgba(200, 200, 200, 0.2)'
                    }}
                />
                <Text style={styles.logoTitle}>{item.navLabel}</Text>
            </View>
        );
    }

    render() {
        return (
            <View style={styles.mainWrapper} 
                onLayout={() => {
                    this.setState({
                        viewport: {
                            width: Dimensions.get('window').width,
                            height: Dimensions.get('window').height
                        }
                    });
                }}>
                <Text style={styles.carouselTitle}>Shop By Fit</Text>
                <Carousel
                    inactiveSlideOpacity={1}
                    inactiveSlideScale={1}
                    firstItem={0}
                    sliderWidth={this.state.viewport.width}
                    itemWidth={this.state.viewport.width / 2.5}
                    activeSlideAlignment={'start'}
                    data={this.state.carouselData[0].visualNavTiles.visualNavTile}
                    renderItem={this.renderItem}
                    enableMomentum={true}
                />
            </View>

        )
    }
}

const styles = StyleSheet.create({
    carouselWrapper: {
        flex: 1,
        flexDirection: 'row',
        alignItems: 'center',
        backgroundColor: '#fff',
        borderRadius: 5,
        borderWidth: 1,
        borderColor: '#000',
        marginRight: 8
    },
    mainWrapper: {
        paddingVertical: 20,
        flex: 1
    },
    carouselTitle: {
        color: '#000',
        fontSize: 20,
        fontWeight: 'bold',
        textAlign: 'center',
        paddingBottom: 15
    },
    logoStyle: {
        width: 50,
        height: 50,
        borderRadius: 10,
    },
    logoTitle: {
        fontSize: 16,
        fontWeight: 'bold',
        textAlign: 'center',
        width: '75%',
        flex: 1, 
        flexWrap: 'wrap'
    }
});