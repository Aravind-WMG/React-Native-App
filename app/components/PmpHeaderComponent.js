import React from 'react';
import { Text, View, StyleSheet, Image, TouchableOpacity } from 'react-native';
import Icon from 'react-native-vector-icons/MaterialIcons';

export class PmpHeaderComponent extends React.Component {
    constructor(props) {
        super(props);
    }
    render() {
        return (
            <View style={styles.navBar}>
                <View style={styles.navBarLeft}>
                    <TouchableOpacity>
                        <Icon name="menu" size={30} color='#000' />
                    </TouchableOpacity>
                    <Image
                        source={require('../images/logo_new.png')}
                        style={styles.imageStyle}
                    />
                </View>
                <View style={styles.navBarRight}>
                    <Text style={styles.userName}>Hi, Daisy</Text>
                    <TouchableOpacity>
                        <Icon name="account-circle" size={40} color='#000' style={styles.accountCircle} />
                    </TouchableOpacity>
                    <TouchableOpacity>
                        <Icon name="shopping-cart" size={35} color='#000' />
                    </TouchableOpacity>
                </View>
            </View>
        )
    }
}
const styles = StyleSheet.create({
    navBar: {
        height: 75,
        flexDirection: 'row',
        alignItems: 'center',
        flex: 1
    },
    navBarLeft: {
        flexDirection: 'row',
        alignItems: 'center',
        justifyContent: 'flex-start',
        paddingLeft: 5
    },
    navBarRight: {
        flexDirection: 'row',
        alignItems: 'center',
        justifyContent: 'flex-end',
        paddingRight: 10
    },
    imageStyle: {
        width: 180,
        height: 27,
        backgroundColor: '#fff',
        marginHorizontal: 20,
    },
    userName: {
        color: '#000',
        fontSize: 16,
        fontWeight: 'bold',
        paddingRight: 5,
    },
    accountCircle: {
        paddingRight: 10,
    }
});