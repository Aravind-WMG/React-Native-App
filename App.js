import React from 'react';
import { StyleSheet, Text, View, ScrollView } from 'react-native';
import {CarouselComponent} from './app/components/CarouselComponent';
import {HeaderComponent} from './app/components/HeaderComponent';
import {SearchComponent} from './app/components/SearchComponent';
import {ProductComponent} from './app/components/ProductComponent';

export default class App extends React.Component {
  render() {
    return (
      <View style={styles.container}>
        <ScrollView>
            <HeaderComponent />
            <SearchComponent />
            <CarouselComponent />
            <ProductComponent/>
          </ScrollView>
      </View>
    );
  }
}

const styles = StyleSheet.create({
  container: {
    flex: 1,
    backgroundColor: '#fff',
  },
});
