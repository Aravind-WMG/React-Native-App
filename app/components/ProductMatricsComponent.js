import React from 'react';
import { Text, View, StyleSheet, ScrollView,Platform } from 'react-native';
import { PmpHeaderComponent } from './PmpHeaderComponent';
import { SearchComponent } from './SearchComponent';
import { ProductComponent } from './ProductComponent';

export class ProductMatricsComponent extends React.Component {
    constructor(props) {
        super(props);
    }
    render() {
        return (
            <View style={styles.container}>
                <ScrollView>
                    <PmpHeaderComponent />
                    <SearchComponent />
                    <ProductComponent nav={this.props.navigation}/>
                </ScrollView>
            </View>
        );
    }
}

const styles = StyleSheet.create({
    container: {
      width:'100%',
      backgroundColor: '#fff',
      marginTop: (Platform.OS === 'ios') ? 20 : 0,
    },
  });