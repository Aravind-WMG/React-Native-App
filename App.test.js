import React from "react";
import App from "./App";
import { PmpHeaderComponent } from "./app/components/PmpHeaderComponent";
import { PmpCarouselComponent } from "./app/components/PmpCarouselComponent";
import { ProductMatricsComponent } from "./app/components/ProductMatricsComponent";
import { Text, View, StyleSheet, ScrollView, Platform } from "react-native";
import renderer from "react-test-renderer";
import Enzyme from "enzyme";
import Adapter from "enzyme-adapter-react-16";
import { shallow } from "enzyme";

Enzyme.configure({ adapter: new Adapter() });

it("PmpCarouselComponent renders correctly", () => {
  const rendered = renderer.create(<PmpCarouselComponent />).toJSON();
  expect(rendered).toMatchSnapshot();
});

it("PmpHeaderComponent renders correctly", () => {
  const rendered = renderer.create(<PmpHeaderComponent />).toJSON();
  expect(rendered).toMatchSnapshot();
});

describe("<ProductMatricsComponent />", () => {
  it("it should render 1 view component", () => {
    const wrapper = shallow(<ProductMatricsComponent />);
    expect(wrapper.find(View).length).toBe(1);
  });

  it("it should render 1 ScrollView", () => {
    const wrapper = shallow(<ProductMatricsComponent />);
    expect(wrapper.find(ScrollView).length).toBe(1);
  });

  it("it should render PmpHeaderComponent", () => {
    const wrapper = shallow(<ProductMatricsComponent />);
    expect(wrapper.find(PmpHeaderComponent).length).toBe(1);
  });
});
