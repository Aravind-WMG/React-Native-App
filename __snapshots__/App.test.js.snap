// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PmpHeaderComponent renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "flexDirection": "row",
      "height": 75,
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "flex": 1,
        "flexDirection": "row",
        "justifyContent": "flex-start",
        "paddingLeft": 5,
        "width": "50%",
      }
    }
  >
    <View
      accessibilityComponentType={undefined}
      accessibilityLabel={undefined}
      accessibilityTraits={undefined}
      accessible={true}
      collapsable={undefined}
      hasTVPreferredFocus={undefined}
      hitSlop={undefined}
      isTVSelectable={true}
      nativeID={undefined}
      onLayout={undefined}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "opacity": 1,
        }
      }
      testID={undefined}
      tvParallaxProperties={undefined}
    >
      <Text
        accessible={true}
        allowFontScaling={false}
        ellipsizeMode="tail"
        style={
          Array [
            Object {
              "color": "#000",
              "fontSize": 30,
            },
            undefined,
            Object {
              "fontFamily": "Material Icons",
              "fontStyle": "normal",
              "fontWeight": "normal",
            },
          ]
        }
      >
        
      </Text>
    </View>
    <Image
      resizeMode="contain"
      source="test-file-stub"
      style={
        Object {
          "alignSelf": "stretch",
          "backgroundColor": "#fff",
          "height": 27,
          "marginHorizontal": 20,
          "width": 160,
        }
      }
    />
  </View>
  <View
    style={
      Object {
        "alignItems": "center",
        "flex": 1,
        "flexDirection": "row",
        "justifyContent": "flex-end",
        "paddingRight": 10,
        "width": "50%",
      }
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#000",
          "fontSize": 16,
          "fontWeight": "bold",
          "paddingRight": 5,
        }
      }
    >
      Hi, Daisy
    </Text>
    <View
      accessibilityComponentType={undefined}
      accessibilityLabel={undefined}
      accessibilityTraits={undefined}
      accessible={true}
      collapsable={undefined}
      hasTVPreferredFocus={undefined}
      hitSlop={undefined}
      isTVSelectable={true}
      nativeID={undefined}
      onLayout={undefined}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "opacity": 1,
        }
      }
      testID={undefined}
      tvParallaxProperties={undefined}
    >
      <Text
        accessible={true}
        allowFontScaling={false}
        ellipsizeMode="tail"
        style={
          Array [
            Object {
              "color": "#000",
              "fontSize": 40,
            },
            Object {
              "paddingRight": 10,
            },
            Object {
              "fontFamily": "Material Icons",
              "fontStyle": "normal",
              "fontWeight": "normal",
            },
          ]
        }
      >
        
      </Text>
    </View>
    <View
      accessibilityComponentType={undefined}
      accessibilityLabel={undefined}
      accessibilityTraits={undefined}
      accessible={true}
      collapsable={undefined}
      hasTVPreferredFocus={undefined}
      hitSlop={undefined}
      isTVSelectable={true}
      nativeID={undefined}
      onLayout={undefined}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "opacity": 1,
        }
      }
      testID={undefined}
      tvParallaxProperties={undefined}
    >
      <Text
        accessible={true}
        allowFontScaling={false}
        ellipsizeMode="tail"
        style={
          Array [
            Object {
              "color": "#000",
              "fontSize": 35,
            },
            undefined,
            Object {
              "fontFamily": "Material Icons",
              "fontStyle": "normal",
              "fontWeight": "normal",
            },
          ]
        }
      >
        
      </Text>
    </View>
  </View>
</View>
`;
